### DISCUSSION TITLE
Sync with gitlab/github project's wiki repository

### DISCUSSION THREAD

**Discussion by addisonschultz** [GitbookIO/community#583] [Feature Requests]
Originally requested here: https://github.com/GitbookIO/integrations/issues/447

> There seems to be no way to sync with a gitlab/github project's wiki. The repository for the wiki is not the same as the main repository. The url is the same but with .wiki.git instead of .git. This applies to both github and gitlab.
> 
> See:
> 
> https://docs.github.com/en/communities/documenting-your-project-with-wikis/adding-or-editing-wiki-pages#cloning-wikis-to-your-computer
> https://docs.gitlab.com/ee/user/project/wiki/#create-or-edit-wiki-pages-locally

**Comment by addisonschultz** [comment]
You can import Wikis: https://docs.gitbook.com/content-editor/import

**Reply by cesarizu-incine** [reply]
@addisonschultz Thanks, this request is about synchronizing, and not only importing. I don't think it should be closed.

**Reply by addisonschultz** [reply]
Apologies - I'll reopen the ticket then!

**Reply by DreadKnight** [reply]
> Apologies - I'll reopen the ticket then!

Yet, the ticket is closed *sigh*, which is super important stuff 🐻

**Comment by DreadKnight** [comment]
I don't see an easy way to add and sync and existing wiki from a GitHub project, which makes GitBook fully useless for me atm, sadly.

In my case, I want to have the wiki from
https://github.com/FreezingMoon/AncientBeast/wiki
to be synced and visible here
https://freezingmoon.gitbook.io/ancientbeast

Instead I get the project's ReadMe and other text files...
I even tried setting up all sorts of links for monorepo.

How do I go about it without doing any back flips?

**Reply by addisonschultz** [reply]
I don’t think it’s possible natively how you’re describing - unless there’s a way to get your wiki’s files in the main repo’s directory

**Reply by DreadKnight** [reply]
> I don’t think it’s possible natively how you’re describing - unless there’s a way to get your wiki’s files in the main repo’s directory

Was afraid of that... there used to be a checkbox for just that inside GitHub's settings, but I guess it got removed...

**Reply by cesarizu** [reply]
I think this should be a setting in _Gitbook_, which instead of using the main repo url, it appends `/wiki` at the end.
